server/routes/bookedtoday/unexpectedArrivals/validation/sexValidation.test.ts

```diff
@@ -0,0 +1,13 @@
+import SexValidation from './sexValidation'
+
+describe('SexValidation', () => {
+  it('Should return error if an option is not selected', () =>
+    expect(SexValidation({})).toEqual([
+      {
+        text: "Select this person's sex",
```

Comment:
new sexValidator created because the validation message is different compared to the one in '/server/middleware/validation/sexValidation.ts'